src/ducks/Studio/timeseries/hooks.js

```diff
@@ -85,7 +85,7 @@ export const useTimeseries = (metrics, settings) => {
       let mergedData = []
 
       metrics.forEach(metric => {
-        const { key, reqMeta } = metric
+        const { key, keyAlias = key, reqMeta } = metric
```

Comment:
`keyAlias` is not needed

Reply:
It will cause error, because 'twitter_followers_7d' does't exist on API. @DmitriVanGuard